### PR TITLE
MINIFICPP-2212 use administrative yield duration instead of onschedul…

### PR DESCRIPTION
**Pull request by martinzink**
…e retry interval

Thank you for submitting a contribution to Apache NiFi - MiNiFi C++.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [ ] Does your PR title start with MINIFICPP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically main)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you updated the NOTICE file?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI results for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: PROCESSORS.md [hunk 1]
```diff
@@ -2728,7 +2728,6 @@ In the list below, the names of required properties appear in bold. Any other pr
...
 | **Max Latency**                      | 10 sec                                                                                          |                  | The maximum time a source will wait to send new events.                                                                                                                                                                                                                                                                         |
-| Connection Retry Interval            | 10 sec                                                                                          |                  | The interval with which a source will try to reconnect to the server.                                                                                                                                                                                                                                                           |
```
Comment (szaszm) [inline PROCESSORS.md:2731]: Why is this removed?

Comment (martinzink) [reply PROCESSORS.md]: Great catch, it seems I was too zealous with my purging. Fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1664/commits/ddacfb2844366f6f53e86664b1ec83cac8e83979

**Review by lordgamez — APPROVED** [review]

**Review by adamdebreceni — APPROVED** [review]

**Review by szaszm — APPROVED** [review]

**Comment by szaszm** [comment]
Could you check the conflict?
